deploy.sh

```diff
@@ -9,12 +9,12 @@ if [ -z $TAG ]; then
     exit
 fi
 
-function gzipped_cp {
-    gzip --stdout dist/$1 | aws s3 cp --acl public-read --cache-control max-age=2592000 --content-encoding gzip --content-type $2 - s3://mapbox/mapbox-gl-js/$TAG/$1
-    echo "upload: dist/$1 to s3://mapbox/mapbox-gl-js/$TAG/$1 (gzipped)"
+function upload {
+    aws s3 cp --acl public-read dist/$1 s3://mapbox-gl-js/$TAG/$1
```

Comment:
afaict s3 guesses the mime types for all these files correctly, but it doesn't hurt to keep this explicit.